controllers/flinkcluster_reconciler.go

```diff
@@ -116,9 +117,16 @@ func (reconciler *ClusterReconciler) reconcileDeployment(
 	}
 
 	if desiredDeployment != nil && observedDeployment != nil {
+		if getUpdateState(reconciler.observed) == UpdateStateUpdating {
```

Comment:
Add corresponding log for other update states e.g., StoppingJob.

Reply:
Done.